ui-tests/cypress/integration/components/ux_fragments.spec.js

```diff
@@ -1,808 +1,665 @@
+                  page.getContent().getFragmentTable()
+                      .should('exist').and('be.visible');
+                  page.getContent().getEditButton()
+                      .should('exist').and('be.visible');
+                  page.getContent().getReferencedUxFragments()
+                      .should('exist').and('be.visible')
+                      .and('contain.text', 'Referenced UX fragments');
+                  page.getContent().getReferencedPageTemplates()
+                      .should('exist').and('be.visible')
+                      .and('contain.text', 'Referenced page templates');
+                  page.getContent().getReferencedWidgetTypes()
+                      .should('exist').and('be.visible')
+                      .and('contain.text', 'Referenced widget types');
+                }));
       });
 
-      it([Tag.SANITY, 'ENG-3522'], 'Adding a fragment', () => {
-        generateRandomFragment().then(fragment => {
-          openFragmentsPage()
-              .then(page =>
-                  page.getContent().openAddFragmentPage());
-
-          cy.get('@currentPage')
-            .then(page => {
-              page.getContent().getCodeInput().then(input => page.getContent().type(input, fragment.code));
-              page.getContent().getGuiCodeInput().then(input => page.getContent().type(input, fragment.guiCode));
-            })
-            .then(page => page.getContent().save())
-            .then(page => {
-              cy.validateToast(page);
-              cy.wrap(fragment).as('fragmentToBeDeleted');
-
-              page.getContent().getTableRows(fragment.code).should('be.visible')
-                  .children(htmlElements.td).then(cells =>
-                  cy.validateListTexts(cells, [fragment.code]));
-            });
-        });
-      });
-      it([Tag.SANITY, 'ENG-3522'], 'Editing a fragment', () => {
-        addRandomFragment().then(fragment => {
-          openFragmentsPage()
-              .then(page =>
-                  page.getContent().getKebabMenu(fragment.code).open().openEdit(fragment.code))
-              .then(page => page.getContent().getGuiCodeInput().then(input => page.getContent().type(input, fragment.guiCode)))
-              .then(page => page.getContent().save())
-              .then(page => {
-                cy.validateToast(page);
-                page.getContent().getTableRow(fragment.code).should('be.visible')
-                    .children(htmlElements.td).then(cells =>
-                    cy.validateListTexts(cells, [fragment.code, false]));
-              });
-        });
-      });
-      it([Tag.SANITY, 'ENG-3522'], 'Cloning', () => {
-        addRandomFragment().then(fragment => {
-          openFragmentsPage()
-              .then(page => {
-                page.getContent().getKebabMenu(fragment.code).open().openClone(fragment.code);
-                cy.wrap(fragment).as('fragmentToBeDeleted');
-              });
-
-          cy.get('@currentPage')
```

Comment:
No need to break and get the currentPage from alias; if needed, return the page after performing the actions wanted